wasm/jit/jit_value_location_amd64.go

```diff
@@ -214,9 +224,9 @@ func (s *valueLocationStack) takeFreeRegister(tp generalPurposeRegisterType) (re
 	var targetRegs []int16
 	switch tp {
 	case gpTypeFloat:
-		targetRegs = gpFloatRegisters
+		targetRegs = generalPurposeFloatRegisters
 	case gpTypeInt:
-		targetRegs = gpIntRegisters
+		targetRegs = unreservedGeneralPurposeIntRegisters
```

Comment:
so that's what gp is? I thought it was "general practitioner" 👨‍⚕️

Reply:
removed all the usage of "gp": https://github.com/tetratelabs/wazero/pull/84/commits/43f965864a047b0c81f9e3c1e6475b24dc79f553 👯